### PR TITLE
Comment unused appender

### PR DESCRIPTION
**Pull request by jtjeferreira**
to avoid https://jira.qos.ch/browse/LOGBACK-925 and having a thread not being stopped

### CODE REVIEW

File: src/main/g8/conf/logback.xml [hunk 1]
```diff
@@ -23,9 +23,11 @@
     </encoder>
   </appender>
 
+  <!--
   <appender name="ASYNCFILE" class="ch.qos.logback.classic.AsyncAppender">
     <appender-ref ref="FILE" />
   </appender>
+  -->
```
Comment (octonato) [inline src/main/g8/conf/logback.xml]: I'm wondering why the ASYNCFILE was commented out. Maybe the idea was to leave it for production only?

**Review by octonato — DISMISSED** [review]

File: src/main/g8/conf/logback.xml (hunk 1 above)
Comment (octonato) [reply src/main/g8/conf/logback.xml]: For the record, the Java seed has it enabled: 
https://github.com/playframework/play-java-seed.g8/blob/2.8.x/src/main/g8/conf/logback.xml#L38

So, maybe better to align it here. By that I mean, we enable the appender in the scala seed. Users can then decided if they want it or not.

Comment (jtjeferreira) [reply src/main/g8/conf/logback.xml]: Sorry for the delay! Fixed...

**Review by octonato — APPROVED** [review]

**Comment by octonato** [comment]
Obrigado João!

**Pull request merged**